tests/phpunit/includes/mock-mailer.php

```diff
@@ -1,8 +1,9 @@
 <?php
 require_once ABSPATH . 'wp-includes/PHPMailer/PHPMailer.php';
 require_once ABSPATH . 'wp-includes/PHPMailer/Exception.php';
+require_once ABSPATH . 'wp-includes/class-wp-phpmailer.php';
 
-class MockPHPMailer extends PHPMailer\PHPMailer\PHPMailer {
+class MockPHPMailer extends WP_PHPMailer {
```

Comment:
Add some class doc. You can check here,

https://github.com/WordPress/wordpress-develop/blob/32880ec4e910837cd267e1685733867b243e22fa/tests/phpunit/includes/testcase-ajax.php#L2-L9